src/_reducers/assetTypes.reducer.js

```diff
@@ -40,12 +40,14 @@ export default (state = initialState, action) => {
     case LOAD_DROPDOWN_ASSET_TYPES_SUCCESS:
       return {
         ...state,
-        assetTypes: [...action.payload]
+        assetTypes: [...action.payload],
```

Comment:
Is there any difference between `action.payload` and `[...action.payload]` ?

Reply:
One has a comma at the end because I added the `isLoading: false`